Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -46,6 +52,49 @@ def setUp(self):
         if error:
             print("error --> %s" % error.decode())
 
+    def test_cohesity_restore_from_last_snapshot(self):
+        """
+        Test the Cohesity_Restore_From_Last_Snapshot playbook.
+        """
+        print("Starting test_cohesity_restore_from_last_snapshot...")
+        playbook_name = "Cohesity_Restore_From_Last_Snapshot"
+        subscription_id = get_subscription_id()
+        result = search_alert_id_in_incident(
+            self.alert_id, self.resource_group, self.workspace_name)
+        jsObj = result[0]
+        incident_id = jsObj["id"].split("/")[-1]
```

Comment:
Could you please add a example format of what's expected in jsObj["id"] here?

Reply:
done